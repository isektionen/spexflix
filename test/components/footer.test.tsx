import React from 'react'
import { render } from '../testUtils'
import Footer from '../../components/footer'

describe('Footer component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(
      <Footer publisher="foo" copyrightFromYear={1900} />,
      {}
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
