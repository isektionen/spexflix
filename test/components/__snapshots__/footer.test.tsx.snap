// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <p>
      YouFlix is made just for fun and is open sourced at 
      <a
        href="https://github.com/vmorsell/react-youflix"
        rel="noreferrer"
        target="_blank"
      >
        GitHub
      </a>
      .
    </p>
    <p>
      Â© 1900-2020 foo.
    </p>
  </footer>
</DocumentFragment>
`;
